specifications/SimplifiedFastPaxos/FastPaxos.cfg

```diff
@@ -0,0 +1,21 @@
+CONSTANTS
+    any = any
+    none = none
+    r1 = r1
+    r2 = r2
+    r3 = r3
+    r4 = r4
+    Replicas = {r1, r2, r3, r4}
+    v1 = v1
+    v2 = v2
+    v3 = v3
+    Values = {v1, v2, v3}
+    Ballots = {0, 1, 2, 3, 4}
+    FastBallots = {1, 3}
+    Quorums = {{r1, r2, r3}, {r1, r2, r4}, {r1, r3, r4}, {r2, r3, r4}}
+    FastQuorums = {{r1, r2, r3}, {r1, r2, r4}, {r1, r3, r4}, {r2, r3, r4}}
+
+SPECIFICATION FastSpec
+INVARIANTS FastTypeOK
+PROPERTY FastSafetyProperty
```

Comment:
Can you also check PaxosSpec, as announced in a theorem in FastPaxos.tla?

Reply:
May I know how I can do this? Is there a way to add 2 specifications to 1 .cfg file?

Reply:
Thank you. I have removed the theorem.